.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
Not a fan of adding this. You want to ignore it because you build the code there not because it is a file that should be ignored. Can you put that in your `.git/info/exclude` instead?

Reply:
OK.